datarepo-clienttests/src/main/java/scripts/testscripts/EnumerateProfiles.java

```diff
@@ -18,14 +18,19 @@ public EnumerateProfiles() {
   }
 
   public void userJourney(TestUserSpecification testUser) throws Exception {
-    ApiClient apiClient = DataRepoUtils.getClientForTestUser(testUser, server);
-    ResourcesApi resourcesApi = new ResourcesApi(apiClient);
-    EnumerateBillingProfileModel profiles = resourcesApi.enumerateProfiles(0, 10);
+    try {
+      ApiClient apiClient = DataRepoUtils.getClientForTestUser(testUser, server);
+      ResourcesApi resourcesApi = new ResourcesApi(apiClient);
+      EnumerateBillingProfileModel profiles = resourcesApi.enumerateProfiles(0, 10);
 
-    int httpStatus = resourcesApi.getApiClient().getStatusCode();
-    logger.debug(
-        "Enumerate profiles: HTTP status {}, number of profiles found = {}",
-        httpStatus,
-        profiles.getTotal());
+      int httpStatus = resourcesApi.getApiClient().getStatusCode();
+      logger.debug(
+          "Enumerate profiles: HTTP status {}, number of profiles found = {}",
+          httpStatus,
+          profiles.getTotal());
+    } catch (Exception e) {
+      logger.info("User journey failed: ", e);
```

Comment:
In general, it's better to log at the catch site, not the throw site. Is this a case where the catch site for this throw doesn't log? Since catch site usually log, logging at a throw will cause exceptions to be logged twice.

Reply:
Yeah, this is probably a change that's required in the test running...all I'm interested in here it printing stuff to the console...right now we don't print anything.  The handler that redirects to the file should probably be changed to also print to the console

Reply:
It might be worth a talk w/ @marikomedlock as we think about how to design the logging to better fit our testing needs. If I remember correctly, we're purposefully suppressing logs for use with actual performance testing. This accounts for the case when we're running with thousands of runs and we don't want the logs to be overwhelmed.